src/api/models/nls/ncn_rebuild.go

```diff
@@ -33,7 +33,8 @@ type CreateRebuildWorkflowRequest struct {
 	Hosts          []string `json:"hosts"`
 	DryRun         bool     `json:"dryRun"`
 	SwitchPassword string   `json:"switchPassword,omitempty"`
-	WipeOsd        bool     `json:"wipeOsd,omitempty"`
+	ZapOsds        bool     `json:"zapOsds,omitempty"`
+	WorkflowType   string   `json:"workflowType,omitempty"`
```

Comment:
I would suggest adding comments on each of these lines so that the description of the request params gets populated with something helpful to the API consumer. E.g.:

```suggestion
	ZapOsds        bool     `json:"zapOsds,omitempty"` // Description of this parameter here
	WorkflowType   string   `json:"workflowType,omitempty"` // Description of this parameter here
```